### DISCUSSION TITLE
Starting `fzf` at a given cursor position?

### DISCUSSION THREAD

**Discussion by Kabouik** [junegunn/fzf#3934] [Q&A]
I am writing a script whereby pressing `right` enters a directory, and `left` does a `cd` to the parent directory and lists `fzf` items with `fd` again. Since `left` actually feels like a "Go back to the parent folder" key after entering a directory, I would like it to remember the position of the directory that was explored in the first place.

Unfortunately, I don't think I can save `FZF_POS` when entering the folder and then use the `pos($FZF_POS)` action when going back because the action would be triggered on a new key press, while in my case when I press `left`, there's a new round of `fd <args> | fzf` before showing the list again.

I have tried `start:pos($FZF_POS)"` (see below), but no luck. Is there a trick I could use?

```bash
main() {
    while :; do
	clear
	cd "$dir"
	selection=$(fd .  \
			| fzf --query "${tmp}" \
                              --bind="start:pos($FZF_POS)" \
			      --bind="left:execute(echo 'cdup' > /tmp/fzplayer-arg)+accept" \
			      --bind="right:execute(echo 'cddown' > /tmp/fzplayer-arg)+accept")

	if [[ -f "/tmp/fzplayer-arg" ]]; then
            arg=$(cat /tmp/fzplayer-arg)
            rm /tmp/fzplayer-arg
        fi

        if ! [[ -v "$selection" ]]; then
            clear
            case "$arg" in
                cddown)
		    # cd into selected directory and relaunch main()
                    dir="$(realpath "$selection")"
		    continue
                    ;;
                cdup)
		    # cd into parent directory and relaunch main()
		    if [[ "$(realpath "$dir")" == "$(realpath "$media")"* ]]; then
			dir="$(dirname "$dir")"
		    fi
                    continue
                    ;;
            esac
        fi
    done
}

**Comment by tzhouhc** [comment]
try bind to `result` instead? same issue for me and `--bind 'result:last'` did work for me.

**Reply by tzhouhc** [reply]
Granted my use case is much simpler and I'm not sure if this would work with the whole file-explorer setup you have
